filesystems/blob/src/test/scala/cromwell/filesystems/blob/AzureFileSystemSpec.scala

```diff
@@ -0,0 +1,28 @@
+package cromwell.filesystems.blob
+
+import com.azure.storage.blob.nio.{AzureFileSystem, AzureFileSystemProvider}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.time.Instant
+import scala.compat.java8.OptionConverters._
+import scala.jdk.CollectionConverters._
+
+class AzureFileSystemSpec extends AnyFlatSpec with Matchers {
+  val now = Instant.now()
+  val container = BlobContainerName("testConainer")
+  val exampleSas = BlobPathBuilderFactorySpec.buildExampleSasToken(now)
+  val exampleConfig = BlobFileSystemManager.buildConfigMap(exampleSas, container)
+  val exampleStorageEndpoint = BlobPathBuilderSpec.buildEndpoint("testStorageAccount")
+  val exampleCombinedEndpoint = BlobFileSystemManager.combinedEnpointContainerUri(exampleStorageEndpoint, container)
+
+
+  // This test is passing locally but not in CI, because it can't find a filesystem provider with the azb scheme.
+  // Some sort of build issue?
```

Comment:
I assume this comment is obsolete, since the test is looking to be checked in?

Reply:
🤦‍♀️

Reply:
Whoops! Will remove as the test is now working in CI.